chandra_aca/star_probs.py

```diff
@@ -227,24 +247,167 @@ def acq_success_prob(date=None, t_ccd=-10.0, mag=10.0, color=0.6, spoiler=False,
     elif model == 'spline':
         probs = spline_model_acq_prob(mags, t_ccds, colors, halfwidths)
 
+    elif model.startswith('grid-'):
+        probs = grid_model_acq_prob(mags, t_ccds, colors, halfwidths, model=model)
+
     else:
-        raise ValueError("`model` parameter must be 'sota' or 'spline'")
+        raise ValueError("`model` parameter must be 'sota' | 'spline' | 'grid-*'")
 
+    # Deal with color=0.7 stars and/or spoiled stars.  The spoiling correction
+    # is a relic that should never be used once proseco is promoted.
```

Comment:
I've already forgotten; how does proseco help with this?

Reply:
Proseco assumes a star acq probability is for an isolated star and then explicitly computes the probability that it is spoiled by something else nearby.

Reply:
Gotcha.  So does that just mean that the acq_success_prob for an individual star should not be computed in chandra_aca when we promote proseco (so we won't need to come back here to deal with the spoiling correction, right?)?  Also still wondering about stuff like "t_ccd_warm_limit" etc which is set for stars and not set to take proseco probabilities.

Reply:
Granted t_ccd_warm_limit will be pretty moot for proseco catalogs anyway, as we'll want to re-selected the stars using the updated background for the increasing temperatures.  I'm thinking that might need to run on hydra too...